include/platform/random.h

```diff
@@ -53,6 +53,18 @@ extern "C" {
  */
 
 /**
+ * Get random stream.
+ *
+ * @param[in]   aInputLength      The expected size of random values.
+ * @param[out]  aOutput           A pointer to the buffer for the generated random stream. The pointer should never be NULL.
+ * @param[out]  aOutputLength     A pointer to the generated size of random stream. The pointer should never be NULL.
```

Comment:
aOutputLength is the size of random generated by RNG. It is supposed to be the same as aInputLength, but maybe less than aInputLength.
Do I understand your question correctly?